devscripts/run_tests.py

```diff
@@ -29,12 +29,12 @@ def run_tests(*tests, pattern=None, ci=False):
     arguments = ['pytest', '-Werror', '--tb=short']
     if ci:
         arguments.append('--color=yes')
+    if pattern:
+        arguments.extend(['-k', pattern])
     if run_core:
-        arguments.extend(['-m', 'not download'])
+        arguments.extend(['--ignore', 'test/test_download.py'])
```

Comment:
need to revert this. There are other tests marked as `download` besides `test/test_download.py`, and our CI is running them now and that's why our core tests are failing all over
```suggestion
        arguments.extend(['-m', 'not download'])
```